traversal/procedure/ProcedureVertex.java

```diff
@@ -132,6 +145,8 @@ public String toString() {
         return str;
     }
 
+    public abstract Forwardable<Vertex<?, ?>, Order.Asc> intersect(Map<ProcedureEdge<?, ?>, Forwardable<Vertex<?, ?>, Order.Asc>> iterators, Traversal.Parameters parameters, Order.Asc order);
```

Comment:
Support overloading the intersection operator.

Made sense at the time as a way to support "partial specifications" of the to-vertex, rather than an exact vertex identity.
e.g. for a virtual/functional vertex such as a ValueVertex, the inputs only partially specify the final identity of the vertex. All the specifications from the input-vertices together uniquely determine the output vertex.

This may be the wrong approach when there are multiple possible output vertices for a given specification.